src/Fhir.Anonymizer.AzureDataFactoryPipeline/scripts/ArmTemplate/arm_template.json

```diff
@@ -106,7 +106,11 @@
                             "folderPath": "[concat(parameters('sourceStorageActivityApplicationContainer'), '/AdfApplication')]",
                             "extendedProperties": {
                                 "sourceConnectionString": "[parameters('sourceStorageLinkedService_connectionString')]",
-                                "destinationConnectionString": "[parameters('destinationStorageLinkedService_connectionString')]"
+                                "destinationConnectionString": "[parameters('destinationStorageLinkedService_connectionString')]",
+                                "skipExistedBlob": {
+                                    "value": "@pipeline().parameters.skipExistedBlob",
+                                    "type": "Expression"
```

Comment:
Where could users specify this parameter in ADF scenario?

Reply:
ADF default support parameters at pipeline level. When customer trigger pipeline they can provide parameters, and if customer us UI, there would be a window pop up for parameter.